lib/admin_page.php

```diff
@@ -20,7 +20,7 @@ function ppi_admin_page_init() {
 function ppi_add_menu_item() {
     add_menu_page(
         'ProPhoto Installer',
-        ppi_p6_is_installed() ? 'P6 Test Drive' : 'P6 Installer',
+        ppi_p7_is_installed() ? 'P7 Test Drive' : 'P7 Installer',
```

Comment:
I thought we were going to use  an actual php namespace? `namespace ppi7` and then just use sensible function names like `is_installed()`